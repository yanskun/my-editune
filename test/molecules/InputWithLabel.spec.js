import { shallowMount } from "@vue/test-utils"
import Component from "~/components/molecules/InputWithLabel.vue"

describe("InputWithLabel", () => {
  test("snapshot text", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        value: "test",
        label: "test label"
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })

  test("snapshot number", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        label: "test label",
        value: 0,
        type: "number"
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
