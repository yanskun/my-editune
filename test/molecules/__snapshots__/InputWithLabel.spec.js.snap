// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWithLabel snapshot number 1`] = `
<div
  class="input-with-label"
>
  <el-input-stub
    autocomplete="off"
    class="input"
    type="number"
    validateevent="true"
  />
  <p
    class="label"
  >
    test label
  </p>
</div>
`;

exports[`InputWithLabel snapshot text 1`] = `
<div
  class="input-with-label"
>
  <el-input-stub
    autocomplete="off"
    class="input"
    type="text"
    validateevent="true"
  />
  <p
    class="label"
  >
    test label
  </p>
</div>
`;
